scripts/fuzzing_ci.sh

```diff
@@ -110,7 +114,8 @@ for i in "${!all_fuzzing_targets[@]}"; do
     echo "Error: Fuzz testing for target ${fuzzing_target} failed."
     script_return=${EXIT_FAILURE}
     echo "Fuzzing logs:"
-    echo "${fuzzing_results}"
+    echo "${fuzzing_results}" | tail -n${LINES_TO_OUTPUT}
```

Comment:
Somewhat related question: The output here is generated by the run_xcode_fuzzing function, which calls xcodebuild. Clearly, it generates stdout, which we assign to this variable. Does it also generate stderr? If so, we might still end up with an overly long log.

Reply:
I'm not sure if it generates stderr -- at the very least, I could see a significant reduction in log length when running locally. I'd prefer to submit as is and do a followup.